examples/vision/edsr.py

```diff
@@ -0,0 +1,276 @@
+"""
+Title: Enhanced Deep Residual Networks for Single Image Super-Resolution
+Author: Gitesh Chawda
+Date created: 26-02-2022
+Last modified: 26-02-2022
+Description: Implementation of Enhanced Deep Residual Networks for Single Image Super-Resolution(EDSR).
+"""
+"""
+This example demonstrates how to implement EDSR paper using keras and tensorflow. 
```

Comment:
We already have a super-resolution example: https://keras.io/examples/vision/super_resolution_sub_pixel/ The introduction here should reference the previous one, should articulate what makes this new method different from the other, and should explain when to use one method over the other.